lib/files/downloader.ts

```diff
@@ -108,6 +106,17 @@ export class Downloader {
                    response.destroy();
                    resolve(false);
                  } else {
+                   if (opt_proxy) {
+                     let pathUrl = url.parse(options.url).pathname;
```

Comment:
But if `fileUrl` contains a query or hash shouldn't that be part of `newFileUrl`?